scripts/test-in-lxd.sh

```diff
@@ -2,34 +2,40 @@
 set -eux
 IMAGE=$1
 SCRIPT=$2
+TESTER=${IMAGE##*:}
 
 lxd init --auto
 
-lxc launch $IMAGE tester
-lxc config device add tester code disk source=`pwd` path=/subiquity
+if [ -z "$(lxc list -f csv | grep $TESTER)" ]
```

Comment:
`grep $TESTER` might catch the wrong name, depending on the name.
What about something like:
```suggestion
if lxc list -f csv | grep -qv "^${TESTER},"
```

Reply:
You can altogether avoid the grep and catch the right instance with something like

if [ -z "$(lxc list -f csv -c n ^${TESTER}$)" ]; then

(probably the $ at the end of the regex must be escaped)

Reply:
Thank you guys for the suggestions. I'll go with @jibel, avoiding grep. It works on bash with and without escaping the last $.